thrift/lib/go/thrift/clientconn.go

```diff
@@ -20,24 +20,33 @@ import (
 	"fmt"
 )
 
-// ClientConn holds all the connection information for a thrift client
-type ClientConn struct {
+type ClientConn interface {
```

Comment:
Yes! Interfaces make the code more flexible, scalable and it’s a way to achieve polymorphism.